src/Collection.ts

```diff
@@ -311,13 +311,16 @@ export function createCollectionMixin<TBase extends Constructor>(Base: TBase) {
      * Given a bounding box, return all the objects of the collection that are contained in the bounding box.
      * If `includeIntersecting` is true, return also the objects that intersect the bounding box as well.
      * This is meant to work with selection. Is not a generic method.
```

Comment:
```suggestion
```

Why can't it be used in general? It is a good piece of logic

Reply:
it was written for intersection on the canvas with the drag rectangle.
We are not promoting it now, promoting is possible without breaking later.